### PR TITLE
Inna styles for login forms

### LINKED ISSUE #75: Styles for login forms

### PR DESCRIPTION
**Pull request by Innapoliakova**
WHAT?

I changed login fields - now when we click login button we see by default one field for login - and two options - login or join (sign up). Also I added close button in the field form. 

WHY? 
Users can chose - to login or to join, and can open field what they need (instead of both fields at the same time). 
CUsers can click close button to close login form field - no page reload.

### CODE REVIEW

File: client/src/components/Header.css [hunk 1]
```diff
@@ -66,7 +66,7 @@ button:hover {
   padding: 20px;
   background-color: #f2f2f2;
   border-radius: 5px;
-  margin: 10px;
+  /* margin: 10px; */
```
Comment (oliverlloyd) [inline client/src/components/Header.css]: You can delete this code - it isn't needed

File: client/src/components/Header.css [hunk 2]
```diff
@@ -91,10 +91,28 @@ button:hover {
...
   font-size: 24px;
   margin-bottom: 20px;
 }
 
+.close-button {
+  position: absolute;
+  top: -190px;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
```
Comment (oliverlloyd) [inline client/src/components/Header.css]: The close button is a `button` element so you don't need to set the cursor. It doesn't make any difference but just so you know!

**Review by MohamedAbdi114 — APPROVED** [review]
looks good

File: client/src/components/upload.css [hunk 3]
```diff
@@ -38,8 +38,8 @@
 
   .submitbtn{
     position: absolute;
-    left: 1100px;
-    top: 41%;
+    /* left: 1100px;
+    top: 41%; */
```
Comment (oliverlloyd) [inline client/src/components/upload.css]: Not needed 😄

File: client/src/pages/Home.css [hunk 4]
```diff
@@ -1,4 +1,4 @@
-.logo {
+/* .logo {
```
Comment (oliverlloyd) [inline client/src/pages/Home.css]: Can be deleted

**Review by oliverlloyd — APPROVED** [review]
This is great work. Perfect use of React to show hide content 👍 

I made some very small comments about deleting unused code but this is solid work

**Pull request merged**